package.json

```diff
@@ -57,10 +57,8 @@
     "sass": "^1.29.0",
     "sass-loader": "^8.0.2",
     "sinon": "^8.1.1",
-    "style-resources-loader": "^1.4.1",
```

Comment:
Yay, thanks for removing unused imports!

Also, please update the app version (on line 3) so we can tell what version is running when deployed.

Reply:
will do